build_tools/build_mlir.sh

```diff
@@ -51,4 +52,4 @@ cmake -GNinja \
   -DCMAKE_BUILD_TYPE=RelWithDebInfo \
   -DLLVM_ENABLE_ASSERTIONS=On
 
-cmake --build "$build_dir" --target all --target mlir-cpu-runner
```

Comment:
Any clue why this was here? Are we able to target mlir-cpu-runner? Will that build a more specific targets that improves build speed?

Reply:
No idea. I think we inherited this from MLIR-HLO build where it was perhaps load-bearing, but we aren't using it in our repo. Also, due to some reason, this target didn't work on my macbook, so I figured that it would be useful to remove it.